src/tests/ftest/util/server_utils.py

```diff
@@ -516,26 +530,48 @@ def clean_files(self):
         clean_cmds = [
             "find /mnt/daos -mindepth 1 -maxdepth 1 -print0 | xargs -0r rm -rf"
         ]
-
-        if self.runner.job.yaml_params.is_nvme() or \
-           self.runner.job.yaml_params.is_scm():
+        if self.runner.job.yaml_params.is_nvme():
             clean_cmds.append("sudo rm -rf /mnt/daos; sudo umount /mnt/daos")
-
+        # scm_mount can be /mnt/daos0 or /mnt/daos1 for two daos_server
+        # instances. Presently, not supported in DAOS. The for loop needs
+        # to be updated in future to handle it. Single instance pmem
+        # device should work now.
+        if self.runner.job.yaml_params.is_scm():
+            scm_mount_count = len(self.runner.job.yaml_params.
+                                  server_params[-1].scm_list.value)
+            for count in range(0, scm_mount_count):
+                clean_cmds.append("sudo umount /mnt/daos; \
```

Comment:
Shouldn't we be conditionally adding the `sudo umount /mnt/daos` command?  Only add the command if it is not already in the `clean_cmds` list - as it may have already been added on line 534.

Reply:
It is a carry-over from previous code... I think it is better to  change it umount scm_mount point... clean_cmds list is just finding only /mnt/daos... It is possible scm_mount point under test.yaml is defined as /mnt/daos0, /mnt/daos1,etc.. I would still keep  the umount for the scm_mount.

Reply:
done.